### PR TITLE
Development Related to EDRD- 4 & 5 Sprints

### PR DESCRIPTION
**Pull request by deepakmulamalla**

### CODE REVIEW

**Review by ckleinacn — APPROVED** [review]

**Comment by deepakmulamalla** [comment]
@cgijeffolsen , @cgiSunilM Can I've this PR reviewed & approved.

Thanking you in advance!

**Comment by deepakmulamalla** [comment]
Hi @cgijeffolsen @RanadheerRG @Nikhiljeenru ,

The latest changes are committed is to fix the Omni studio package version. Currently Repo package version is v246.7 and in environment we have is v248.2. To fix this I've committed One Apex class and related omnistudio components. Please let me know if you have anything.

CC: @ckleinacn

**Comment by deepakmulamalla** [comment]
Hi @cgijeffolsen @RanadheerRG @Nikhiljeenru ,

The latest changes are committed is to update the Node js v16 to Node js v20. As per latest Salesforce update. Committed those changes based on Natalia's Feedback.

CC: @ckleinacn

**Comment by deepakmulamalla** [comment]
Hi CGI Team,

Any Update on PR?

File: dev-app-post/main/default/profiles/Admin.profile-meta.xml
Comment (RanadheerRG) [inline dev-app-post/main/default/profiles/Admin.profile-meta.xml:1]: We are restricted from having additional internal profiles without approval from the Ministry

**Review by RanadheerRG — APPROVED** [review]
@deepakmulamalla  Approved

**Review by Nikhiljeenru — APPROVED** [review]
Approved

**Pull request merged**